gh_actions_exporter/main.py

```diff
@@ -20,6 +21,11 @@ def metrics() -> Metrics:
     return Metrics(get_settings())
 
 
+@lru_cache()
+def github_client() -> GithubClient:
+    return GithubClient(get_settings())
+
+
```

Comment:
Do we need to add a class to return the github client? I feel like the GitHub class is already the one we want.
```suggestion
@lru_cache()
def github_client() -> GitHub:
    settings: Settings = get_settings()

    return GitHub(
        AppInstallationAuthStrategy(
            settings.github_app_id,
            settings.github_app_private_key.get_secret_value(),
            settings.github_app_installation_id
        )
    )
```

Reply:
You're right! I change this